src/apps/backend/modules/task/rest-api/task-router.ts

```diff
@@ -15,5 +15,6 @@ export default class TaskRouter extends ApplicationRouter {
     router.get('/:id', ctrl.getTask);
     router.patch('/:id', ctrl.updateTask);
     router.delete('/:id', ctrl.deleteTask);
+    router.patch('/:id/share', ctrl.shareTask);
```

Comment:
we don't need this api, we can include `ShareTaskParams` in `UpdateTaskParams` and can update the task using the existing patch api